ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -362,13 +362,16 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
                      NULL,          /* Optional name or fabric to resolve       */
                      NULL,          /* Optional service name or port to request */
                      0ULL,          /* Optional flag                            */
-                     hints,        /* In: Hints to filter providers            */
+                     hints,         /* In: Hints to filter providers            */
                      &providers);   /* Out: List of matching providers          */
-    if (0 != ret) {
+    if (FI_ENODATA == -ret) {
+        // It is not an error if no information is returned.
```

Comment:
minor:  c++ comment style

Reply:
We allow this in Open MPI (since compiling Open MPI *requires* a C99 compiler, which officially standardized //-style comments in C).